ios/ReactViewPagerManager.m

```diff
@@ -34,54 +32,30 @@ - (void) goToPage
     }];
 }
 
-- (void) changeScrollEnabled
```

Comment:
why it has been removed ?

Reply:
Defined it inline, as opposed to a separate function.  I think it might have occurred during my Swift -> Objective C conversion?

I can extract it again, if you prefer.

Reply:
It can be defined inline